<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mode RN Deadline Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Desktop/Mode Logo super simple.png</normaloff>../../../../Desktop/Mode Logo super simple.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_AcceptedOffer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Accepted offer date:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDateEdit" name="dateEdit_AcceptedOffer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_MakeAOToday">
          <property name="text">
           <string>Today</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ResetDefaults">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Generat_Report">
          <property name="text">
           <string>Generate Report</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="selectionMode">
         <enum>QCalendarWidget::NoSelection</enum>
        </property>
        <property name="verticalHeaderFormat">
         <enum>QCalendarWidget::NoVerticalHeader</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="11" column="0">
         <widget class="QPushButton" name="pushButton_ShowLess">
          <property name="text">
           <string>Show Less</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_Blank">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="5">
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="4">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="6">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="6">
         <widget class="QLabel" name="label_LongDateCust5">
          <property name="text">
           <string>Full Date</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QSpinBox" name="spinBox_DaysCust2">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>-365</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
        <item row="8" column="5">
         <widget class="QDateEdit" name="custom4_dateEdit">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QSpinBox" name="spinBox_DaysCust1">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>-365</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLabel" name="label_LongDateCust1">
          <property name="text">
           <string>Full Date</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QDateEdit" name="custom2_dateEdit">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QSpinBox" name="spinBox_DaysCust3">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>-365</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
        <item row="7" column="5">
         <widget class="QSpinBox" name="spinBox_DaysCust4">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>-365</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_CustomDate">
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QDateEdit" name="custom1_dateEdit">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QDateEdit" name="custom3_dateEdit">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QCheckBox" name="checkBox_1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="10" column="3">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_CustomDescription">
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
        <item row="5" column="6">
         <widget class="QLineEdit" name="lineEdit_Custom5"/>
        </item>
        <item row="3" column="5">
         <widget class="QLabel" name="label_LongAppraisalDate">
          <property name="text">
           <string>Full Date</string>
          </property>
         </widget>
        </item>
        <item row="9" column="4">
         <widget class="QLabel" name="label_LongDateCust3">
          <property name="text">
           <string>Full Date</string>
          </property>
         </widget>
        </item>
        <item row="9" column="5">
         <widget class="QLabel" name="label_LongDateCust4">
          <property name="text">
           <string>Full Date</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QDateEdit" name="inspection_dateEdit">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="9" column="3">
         <widget class="QLabel" name="label_LongDateCust2">
          <property name="text">
           <string>Full Date</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_LongClosingDate">
          <property name="text">
           <string>Full Date</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="label_LongEMoneyDate">
          <property name="text">
           <string>Full Date</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_ShowMore">
          <property name="text">
           <string>Show More</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6" colspan="2">
         <widget class="QDateEdit" name="financing_dateEdit">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_CustomDays">
          <property name="text">
           <string>Number of Days</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDateEdit" name="earnestMoney_dateEdit">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="3" column="6" colspan="2">
         <widget class="QLabel" name="label_LongFinancingDate">
          <property name="text">
           <string>Full Date</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_LongInspDate">
          <property name="text">
           <string>Full Date</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QDateEdit" name="appraisal_dateEdit">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QDateEdit" name="dateEdit_CalculatedDate">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_EMoney">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Earnest Money</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QSpinBox" name="daysForAppraisal_spinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>-365</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
        <item row="1" column="6" colspan="2">
         <widget class="QSpinBox" name="daysForFinaincing_spinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>-365</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_Date">
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_Inspection">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Inspection</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_Closing">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Closing</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QSpinBox" name="daysForInspection_spinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>-365</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_Apraisal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Appraisal</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6" colspan="2">
         <widget class="QLabel" name="label_financing">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Financing</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinBox_NumOfDays">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>-365</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_NumDays">
          <property name="text">
           <string>Number of Days </string>
          </property>
         </widget>
        </item>
        <item row="8" column="6">
         <widget class="QDateEdit" name="custom5_dateEdit">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="daysForEarnestMoney_spinBox">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>-365</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
        <item row="7" column="6">
         <widget class="QSpinBox" name="spinBox_DaysCust5">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>-365</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_AOorClosing">
          <property name="text">
           <string>AO or Closing</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QComboBox" name="comboBox_Custom1"/>
        </item>
        <item row="5" column="3">
         <widget class="QComboBox" name="comboBox_Custom2"/>
        </item>
        <item row="5" column="4">
         <widget class="QComboBox" name="comboBox_Custom3"/>
        </item>
        <item row="5" column="5">
         <widget class="QComboBox" name="comboBox_Custom4"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
